player/public/js/ekstepRendererApi.js

```diff
@@ -1061,5 +1061,15 @@ window.EkstepRendererAPI = {
         } else {
             console.warn('Invalid Content')
         }
+    },
+
+    /**
+    * This method is used for stop audio play 
+    * when user click outside of the preview area.
+    **/
+    clearPlayer: function() {
+        AudioManager.stopAll();
```

Comment:
How do you handle for the userSwitch ? What if the userSwitch is opend?

Reply:
Same use case for menu also.

How will you handle when menu is opened and suddenly preview is got hide. What if i open the preview for next time will you show the opened menu ? Have you tested this?

Reply:
Change the funciton name as resetPlayer.